### DISCUSSION TITLE
2022-12-28 Weekly meeting 88 Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#327] [General]
Weekly Meeting 88 is happening on Wednesday, December 28, 2022 from 9:00 AM PST(17:00 UTC) - 10:00 AM PST (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

### Discussions: 

- OMC 

- This is a time to build. What can we build?
  - Adding OMI to the Metaverse Street Jin demoed
  - M3 recently started a shop for wearables. This is a great example of things we could be doing and participating in. 
  - Let's build right now. 

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo (December 7!)
- `IPSME` and Me
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
## Demos today 

- Evo has a demo for us today! 
- The Ethereal Engine team released their webxr golf game!

**Reply by mrmetaverse** [reply]
### and beyond!

**Reply by mrhegemon** [reply]
Just released, currently capped at 12 ppl. https://tee.puttclub.io
In testing for NYE: https://dev.etherealengine.com/location/club-ctrl

**Reply by mrmetaverse** [reply]
# Demo I: A multiplayer VR Golf game using webxr! 

![image](https://user-images.githubusercontent.com/63426722/209851602-8a5bb9e7-4607-4ba2-81de-35e17b6fb764.png)

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/209851775-26652ea2-a971-4f3d-9d6c-fbf9c5956c5e.png)

![image](https://user-images.githubusercontent.com/63426722/209851886-43f97df9-081d-4e32-bc31-6cb5142f6c69.png)

![image](https://user-images.githubusercontent.com/63426722/209851900-65099db2-1041-41bd-9135-e8457ebc9a43.png)

Has two virtual machines running the machine through Ethereal Engine: 

![image](https://user-images.githubusercontent.com/63426722/209851952-7eb2d2cf-e77d-4ee1-83c7-f91e2aae6e6f.png)

![image](https://user-images.githubusercontent.com/63426722/209851924-56b1ce5f-6d33-49e9-a479-c2a45dbe893e.png)

**Reply by mrmetaverse** [reply]
The goal is to host large parties on this island, with everyone playing as their own avatars. 

![image](https://user-images.githubusercontent.com/63426722/209852048-ce0e215c-c7b2-4876-b50e-449bc5a2c304.png)

Driven by the underlying gLTF structure. 

Intended to be a test-bed for interoperability. 

![image](https://user-images.githubusercontent.com/63426722/209852133-59e090cc-18ef-444f-9112-c126d349d690.png)

![image](https://user-images.githubusercontent.com/63426722/209852160-6160b7b5-87d7-46cd-b044-fb074d216150.png)

avatar integrations, wallet integrations, modular and open source.

**Reply by mrmetaverse** [reply]
"The number one thing we need to move our physics forward is to implement, and give feedback of our protocols."

**Reply by mrmetaverse** [reply]
# Demo II: Evo and Avatar Joy in Simulacra 

![image](https://user-images.githubusercontent.com/63426722/209852553-e5b41b51-6c9e-4600-81e1-be0c91200c64.png)

![image](https://user-images.githubusercontent.com/63426722/209852608-b91e3b12-e1ad-4b96-adc5-e42163fa85c2.png)

![image](https://user-images.githubusercontent.com/63426722/209852656-4a20524e-da8a-4b21-901b-a1f734aae8ae.png)

It has Social and Chat! 

Great way to introduce people to metaverse concepts in a relatively fun and social way.

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/209852924-676ce940-b56f-40f4-93fc-9aba1a6ad498.png)

**Reply by mrmetaverse** [reply]
contact Rem5 if you have questions about what they are going to do!

**Reply by codefrau** [reply]
![snowglobe](https://user-images.githubusercontent.com/733388/209852503-d6d5a28a-74e8-4a03-915d-de2c764d663e.jpg)

Croquet World: https://croquet.io/snowglobe/

**Reply by mrmetaverse** [reply]
## Demo III: Snowglobe in Croquet!

![image](https://user-images.githubusercontent.com/63426722/209853305-e364a7c9-4d56-4250-990c-0dca57fabe42.png)

![image](https://user-images.githubusercontent.com/63426722/209853374-3c2b9996-13b0-4556-bda9-2a87f66c0fd9.png)

Using microverse and Croquet makes everything client side, no server code involved, multiplayer, and virtually no restrictions.

**Reply by mrmetaverse** [reply]
![image](https://user-images.githubusercontent.com/63426722/209853520-f714689f-dfa1-4aa9-8200-7acd9d606df5.png)

**Comment by mrmetaverse** [comment]
## MSF Delegates 
- biggest announcement was shared last week re: fee proposals 
- #326

**Comment by mrmetaverse** [comment]
# gltf extensions group 

- Had a discussion last week. Major update: Convex hulls stored as a mesh. 
- Liam and Ethereal engine are looking to collab  on the same extension. 
- :point_up: that's what it's all about! sharing learnings and exploring options together.

**Reply by mrmetaverse** [reply]
reaching out to Microsoft to converge on a common approach. 

Very easy to implement, very flexible.

**Comment by mrmetaverse** [comment]
# M3 

Doing a portal crawl on Friday. 

https://github.com/M3-org/research/issues/15

- [Portal Hub](https://hyperfy.io/portal-hub)

**Reply by mrmetaverse** [reply]
Evo: we will document this effort and generate some great content for the website

**Comment by mrmetaverse** [comment]
# Website 

If you are creating content, we would love to add it to our playlists and help get the word out. Create content in January, it will help all of us in February.

**Reply by mrmetaverse** [reply]
We just ask that it be about `open` `metaverse` `interoperability` :+1:

**Comment by mrmetaverse** [comment]
# Events pro tem 

There is a new discord channel for discussing events, like OMC. Pop on in!

**Reply by mrmetaverse** [reply]
OMC: 

## 3 major updates from last week: 

- We can host watch parties. If you have a virtual world and want to host a OMC watch party, reach out! 
- We get various ticket discounts: advanced early bird, etc. 
- We get a booth a the physical conference in L.A. 
  - three use cases
    - We can use the booth as OMI and talk about OMI and have a montage of all of the projects in our community, et al
    - If we were well-managed about it we could do multiple companies and demos (like 3 sections per day) 
    - We can elect as OMI to give the space to one of the companies
